subprojects/testfx-core/src/main/java/org/testfx/util/DebugUtils.java

```diff
@@ -16,31 +16,48 @@
  */
 package org.testfx.util;
 
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Set;
 import java.util.function.Function;
+import java.util.function.Supplier;
 
 import javafx.event.Event;
+import javafx.geometry.Bounds;
+import javafx.geometry.Rectangle2D;
 import javafx.scene.Node;
+import javafx.scene.image.Image;
 import javafx.scene.input.KeyCode;
 import javafx.scene.input.MouseButton;
+import javafx.stage.Screen;
+import javafx.stage.Window;
 
 import org.hamcrest.Matcher;
 import org.testfx.api.FxRobot;
+import org.testfx.api.FxService;
 import org.testfx.api.FxToolkit;
+import org.testfx.service.support.CaptureSupport;
 import org.testfx.service.support.FiredEvents;
 
 /**
- * Utility class for displaying additional info or running code when a test fails using
- * {@link org.testfx.api.FxAssert#verifyThat(Node, Matcher)} or its related methods.
+ * Utility class for displaying additional info, running code, or capturing an image of the test when
+ * a test fails using {@link org.testfx.api.FxAssert#verifyThat(Node, Matcher)} or its related methods.
  *
  * <p>
  *     Quickly chain things together using {@link #compose(Function[])} or use standard handlers like
  *     {@link #informedErrorMessage(String, boolean, FxRobot, boolean, boolean)}. Any {@code indent} parameters
  *     in methods are there to specify what to insert to offset values. Default indent value is three spaces
  *     (e.g. {@code "   "}).
  * </p>
+ * <p>
+ *     When a test fails, an image of the it can be captured and saved to a PNG file. Supports the size
+ *     of a screenshot, a window, an area of the screen (i.e. bounds), or an individual node. Use the
```

Comment:
Also typo with "Supports the of a screenshot" - I assume you accidentley left out "taking" as in "Supports the taking of a screenshot"

Reply:
Actually, that was supposed to indicate what sizes of images it could take. I've reworded it to better clarify that.